src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -108,10 +108,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }))
-
```

Comment:
Apologies for the nitpick, but could you remove this change? Pure whitespace changes can potentially cause conflicts, we try to keep those separate from other changes.